src/master-playlist-controller.js

```diff
@@ -767,7 +767,13 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // code in video.js but is required because play() must be invoked
     // *after* the media source has opened.
     if (this.tech_.autoplay()) {
-      this.tech_.play();
+      const playPromise = this.tech_.play();
+
+      // Catch/silence error when a pause interrupts a play request
+      // on browsers which return a promise
+      if (playPromise !== undefined && typeof playPromise.then !== 'undefined') {
```

Comment:
Thanks for the update @grobolom ! Though in this case the typeof comparing to a function is OK (and preferred), but the `playPromise !== undefined` should probably be `typeof playPromise !== 'undefined'`

Reply:
aaaah. I see. My bad, I'll fix that up!